modules/microshift-config-yaml.adoc

```diff
@@ -0,0 +1,78 @@
+// Module included in the following assemblies:
+//
+// * microshift/using-config-tools.adoc
+
+:_content-type: CONCEPT
+[id="microshift-config-yaml_{context}"]
+= Using YAML configuration
+
+Configure your {product-title} using a YAML file. Values specified in your configuration file are used when you execute a command.
+// Q: can these values be overriden with CLI flags?
+
+== Default Settings
+If you do not create a config.yaml, the following default values will be used.
+// Q: what creates this default yaml?
+// Q: can default file be modified? (instead of making a new one)
++
+.Default YAML file example
+[source,yaml]
+----
+cluster:
+  clusterCIDR: 10.42.0.0/16
+  serviceCIDR: 10.43.0.0/16
+  serviceNodePortRange: 30000-32767
+  dns: 10.43.0.10
+  domain: cluster.local
+  url: https://127.0.0.1:6443
+nodeIP: ""
+nodeName: ""
+logVLevel: 0
+----
+
+[IMPORTANT]
+====
+The configuration file must be located at `~/.microshift/config.yaml` (user-specific) and `/etc/microshift/config.yaml` (system-wide). An existing user-specific `config.yaml` takes precedence.
```

Comment:
- Are there two config files, or could there be just one?
- Can this be reworded to avoid the use of parens? Maybe along the lines of "a user-specific file in `~/.microshift/config.yaml`"